eZ/Bundle/EzPublishDebugBundle/Collector/SiteAccessCollector.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
```

Comment:
Actually, @andrerom only meant this particular line (the `@version` one), not the whole block. Can you restore it please ?